static/app/views/performance/browser/resources/index.tsx

```diff
@@ -75,8 +105,14 @@ function ResourcesLandingPage() {
             <DomainSelector value={filters[SPAN_DOMAIN] || ''} />
             <ResourceTypeSelector value={filters[RESOURCE_TYPE] || ''} />
             <PageSelector value={filters[TRANSACTION] || ''} />
+            <SwitchContainer>
+              <SwitchButton
+                isActive={filters[RESOURCE_RENDER_BLOCKING_STATUS] === 'blocking'}
+                toggle={handleBlockingToggle}
+              />
+              Render Blocking
```

Comment:
needs to be translated.

I'm still 50/50 on the toggle UX, but lets ship this now and get design/product help later.

Reply:
Yeah agreed!